.github/workflows/k8s-pr-test.yml

```diff
@@ -0,0 +1,78 @@
+# About:
+# This action runs when a non-doc Pull Request has been made to the
+# stack directory or any of it subdirectories. This action will test
+# in the same manner the Addon Service applies K8s 1-clicks,
+# using the deploy.sh script and relaying on rollout status to
+# confirm a successfull deploy. This action does NOT test
+# functionality in the k8 1-click app.
+
+# Note:
+# A DigitalOcean token needs to be saved in the repo 
+# Settings --> Secrets with the name, "DIGITALOCEAN_ACCESS_TOKEN"
+# DigitalOcean Tokens doc: 
+#   https://www.digitalocean.com/docs/apis-clis/api/create-personal-access-token/
+# 
+# A 2vCPU 2GB Memory 3 Node cluster in SFO2 is spun up to test. The cluster
+# name includes the Pull Request number. 
+# Once the rollout finishes the cluster is deleted.
+# 
+# This action also makes use of the DOCTL Action:
+#   https://github.com/digitalocean/action-doctl
+
+name: DigitalOcean Kubernetes Test
+on:
+  pull_request:
+    # Do not test if the only update was a doc update,
+    # ie new or edit of an existing markdown file.
+    # Only test if a change was made to the stacks subdirectories.
+    # This assumes any change to source would have to be rendered.
+    paths:
+    - 'stacks/**'    
+    - '!stacks/**/*.md'
+
+jobs:
+
+  build:
+    name: Identify Addon Slug and K8s Rollout Test
+    runs-on: ubuntu-latest
+    steps:
+
+      # Leverage the GitHub API was easier than diffing aginast how actions checkouts out the PR branch.
+      # jq and cut used to save just the Addon Slug string.
+    - name: Identify and set the Addon Slug as an environmental variable
+      id: vars
+      run: echo ::set-env name=ADDON_SLUG::"`curl -sS https://api.github.com/repos/arosales/marketplace-kubernetes/pulls/${{ github.event.pull_request.number }}/files |  jq  -r '.[] | .filename' | cut -d/ -f2`"
```

Comment:
I think you want this to be `https://api.github.com/repos/digitalocean/marketplace-kubernetes/pulls` otherwise it'll just be scoped to `arosales` prs.

Reply:
yeah, if I change the url to `scottcrawford03` I get a not found error because I have no open prs. Updating to `digitalocean` should work.

Reply:
Your correct, and nice catch. This is an artifact left over from my testing. I need to replace `arosales` with `digitalocean`.